### ISSUE TITLE
jscs --fix fails to fix some simple indentation issues

### ISSUE THREAD

**Issue by mvolkmann** [jscs-dev/node-jscs#2143]
The code below is not changed when "jscs --fix" is run on it.
My .jscsrc file has "validateIndentation" set to 2.

````
 function foo(x, y) { // This whole function is indented by one space. It should be changed to two.
   return x + y ;
 }

console.log(foo('hello, ',
       'world!')) ; // this should be changed to two-space indentation
````

My .jscsrc file is attached. I had to make a copy named jscsrc.txt in order to attach it.
[jscsrc.txt](https://github.com/jscs-dev/node-jscs/files/141007/jscsrc.txt)

**Comment by m-abs** [comment]
I can't get "jscs --fix" to work with this .jscsrc. It just outputs the list of style errors.
In jscs@2.1.0 --fix works just fine but in jscs@2.2.0, it doesn't work anymore.

[jscsrc.txt](https://github.com/jscs-dev/node-jscs/files/154490/jscsrc.txt)

**Comment by markelog** [comment]
At this point only major and CST related bugs will be fixed.
